### ISSUE TITLE
Dreamcast buttons Z & C

### ISSUE THREAD

**Issue by MoeFoh** [flyinghead/flycast#966]
Latest Android master version. Phone = Android 11. For the life of me, I can't get C & Z to output when those buttons are binded to a button on the controller. Not a big deal, just experimenting. Something like Rumble Fish 2 (megavolt's gdi) normally responds to lt, rt & c as the same function. Flycast only rt.

Enhancement: option to display button presses, thumbstick & trigger values on the game screen.

**Comment by flyinghead** [comment]
The default dreamcast controller doesn't have C and Z buttons so you have to select a different controller such as Ascii Stick.

**Comment by MoeFoh** [comment]
![ASCIIStickFTWhite](https://user-images.githubusercontent.com/96657677/223526301-3e75a916-8f31-4e2a-bf24-9a292202762e.jpg)

Cool. Thanks. I see twin stick support there, too.

**Comment by MoeFoh** [comment]
How does twin stick configuration work/setup? The twin stick is a 2 d-pad device, yes?

**Comment by flyinghead** [comment]
The twin stick has 2 DPads and X Y A B D and Start buttons. It doesn't have C & Z buttons.

**Comment by MoeFoh** [comment]
Do I map the hardware device mouse to the 2nd d-pad? Just a general
configuration question.

**Comment by flyinghead** [comment]
No, that wouldn't work. You can map your right analog stick to the DPad2, or use the ABXY buttons.
